apps/dashboard/app/views/shared/_welcome.html.erb

```diff
@@ -1 +1 @@
-<%= t('dashboard.welcome_html', logo_img_tag: @user_configuration.disable_dashboard_logo ? "" : logo_image_tag(@user_configuration.dashboard_logo)) %>
+<%= t('dashboard.welcome_html', logo_img_tag: @user_configuration.disable_dashboard_logo ? "" : logo_image_tag(@user_configuration.dashboard_logo)) %>
```

Comment:
Can you re-add this newline here? I know it's a simple thing/change but lots of linters complain about files not ending with newlines.